defaults/main.yml

```diff
@@ -29,6 +29,10 @@ dkim_admin_email: "{{ admin_email | default ( omit ) }}"
 ## Whether opendkim uses the same key for all domains or one key per domain
 dkim_same_key: true
 
+## When generating key with opendkim-gkeygen, default RSA keylen is 2048
+## Other currently possible keylengths: 1024 or 4096.
+dkim_rsa_keylen: 2048
+
 ## Time, in seconds, that the role will pause to show the publik key DNS records
 dkim_dns_record_pause: 0
```

Comment:
Should be removed

Reply:
Hi - but we need a default value for `dkim_rsa_keylen` that will be passed explicitly in `tasks/opendkim_keys.yml`? https://github.com/FoxyRoles/ansible-dkim/pull/12/commits/3ff753a720f89aca668877d55ed56ec677068d5a#diff-c294f4b7ddfeed821300b32372e6be5f3cbe2ae96ad4d4a7a10b46b41043cb09

Reply:
I was talking about `dkim_dns_record_pause` :)

Reply:
ohh! done :)